eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/common/operations/OperationsTestExecutor.java

```diff
@@ -99,6 +102,10 @@ private enum Operation {
               "operations/execution_payload",
               new OperationsTestExecutor<>(
                   "execution_payload.ssz_snappy", Operation.EXECUTION_PAYLOAD))
+          .put(
+              "operations/bls_to_execution_change",
+              new OperationsTestExecutor<>(
+                  "address_change.ssz_snappy", Operation.EXECUTION_PAYLOAD))
```

Comment:
Wrong operation. :)

Reply:
oh that'd make it hard! can't guess what I copied!